.eslintignore

```diff
@@ -0,0 +1,7 @@
+build
```

Comment:
I think you want to end each of those with `/` in order to not match anywhere in the path. At least I think that's the case for .gitignore.
Perhaps start with `/`

Reply:
From the [`eslintignore` docs](https://eslint.org/docs/user-guide/configuring/ignoring-code#ignorepatterns-in-config-files):

> If a glob pattern starts with /, the pattern is relative to the base directory of the config file. For example, /foo.js in lib/.eslintrc.json matches to lib/foo.js but not lib/subdir/foo.js.

so, ah, I think some of these should start with `/` but not all